komodor/manifest.json

```
@@ -0,0 +1,54 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "995fe904-e761-4f2f-8dbf-148baf3f080a",
+  "app_id": "komodor",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "setup": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Track changes across your entire K8s landscape and stack",
+    "title": "Komodor Automation",
+    "media": [
+	 {
+           "media_type": "image",
+	   "caption": "Main services screen.",
+	   "image_url": "./images/komodor_screen_01.png"
+	 },
+	 {
+	   "media_type": "image",
+           "caption": "Service view events timeline and releated services.",
+           "image_url": "./images/komodor_screen_02.png"
+	 },
+         {
+	   "media_type": "image",
+	   "caption": "Services view reviewing a deployment and it's changes.",
+	   "image_url": "./komodor/images/komodor_screen_03.png"
```

I think this needs to be:

```suggestion
	   "image_url": "images/komodor_screen_03.png"
```